iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -384,6 +407,50 @@ public Builder setOnConfigurationListener(
             return this;
         }
 
+        /**
+         * Add a listener to receive configuration changes sent to the device from Cloud IoT
```

Comment:
This comments should be about command instead of configuration.

Reply:
:+1: